docs/source/intro/platform.md

```diff
@@ -1,99 +1,107 @@
+
+You can deploy Apollo Server to any hosted or serverless environment. It supports a variety of popular [Node.js middleware](https://www.apollographql.com/docs/apollo-server/integrations/middleware/).
+
+And if you want your data graph to use a [federated architecture](#federate-your-graph-with-apollo-federation), Apollo Server includes extension libraries that enable it to act as every component of that architecture.
+
+### Build incrementally
+
+Your data graph doesn't immediately need to connect _all_ of your back-end data sources or handle _all_ of your client requests. The Apollo platform supports (and encourages) **incremental adoption**.
+
+As you connect more data sources and expand your schema, Apollo Server can handle a larger and larger percentage of your client data requests. Clients can continue using an existing solution for requests that your data graph doesn't yet support.
+
+## Query your graph with Apollo Client
+
+After you deploy a first version of your data graph, application clients can begin querying it. To execute these queries, you can use [**Apollo Client**](https://www.apollographql.com/docs/react/).
+
+Apollo Client is a customizable, [open-source](https://github.com/apollographql/apollo-client) JavaScript GraphQL client with powerful caching and state management features. It enables developers to define queries directly within the UI components that use them, and automatically update those components as query results arrive or change.
+
+Apollo Client's cache locally replicates the parts of your data graph that your client cares about. This enables your client to query _itself_ for data if it's already present, dramatically improving performance by preventing unnecessary network requests.
+
+### Supported platforms
+
+Apollo Client includes official support for React, and there are community-maintained libraries for [other popular view layers](https://www.apollographql.com/docs/react/integrations/integrations/).
+
+A Swift implementation of Apollo Client is officially supported for [iOS](https://www.apollographql.com/docs/ios/), and there is a community-maintained [Android library](https://www.apollographql.com/docs/android/).
+
+## Manage your graph with Apollo Graph Manager
+
+In addition to its open-source libraries, the Apollo platform provides a cloud-hosted collection of tools that help you measure your graph's performance and grow it safely. These tools are together known as [**Apollo Graph Manager**](https://www.apollographql.com/docs/graph-manager/).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/yZ3_Yvlmy78" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+#### Graph Manager provides the following features to all Apollo users for free:
+
+* A [GraphQL schema registry](https://www.apollographql.com/docs/graph-manager/schema-registry/) that tracks changes
```

Comment:
It's not clear from this section why I would want to push my schema to the registry. We want users to understand that from day 1.